recipes/r-rmagic/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set version = '1.4.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rmagic
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/Rmagic_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/Rmagic/Rmagic_{{ version }}.tar.gz
+  sha256: c0bf24e08866039d8cc07e7d1dfd4767be07cf1a371d6e84123f797013fbc275
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{posix}}zip               # [win]
+  host:
+    - r-base
+    - r-matrix >=1.2_0
+    - r-ggplot2
+    - r-reticulate >=1.4
+  run:
+    - r-base
+    - r-matrix >=1.2_0
+    - r-ggplot2
+    - r-reticulate >=1.4
+
+test:
+  commands:
+    - $R -e "library('Rmagic')"           # [not win]
+    - "\"%R%\" -e \"library('Rmagic')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=Rmagic
+  license: GPL-2
+  summary: 'MAGIC (Markov affinity-based graph imputation of cells) is a method for addressing
+    technical noise in single-cell data, including under-sampling of mRNA molecules,
+    often termed "dropout" which can severely obscure important gene-gene relationships.
+    MAGIC shares information across similar cells, via data diffusion, to denoise the
+    cell count matrix and fill in missing transcripts. Read more: van Dijk et al. (2018)
+    <DOI:10.1016/j.cell.2018.05.061>.'
+  license_family: GPL2
```

Comment:
and why is the generator not adding `license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'`